recipes/python/assets/create_oracle_asset.py

```diff
@@ -0,0 +1,247 @@
+    if credentialName != '':
+        rman_catalog_create_request['data']['attributes']['parameters']['objectList'][0]['asset']['commonAssetAttributes']['credentials']= [{"credentialName": credentialName}]
+    oracle_create_rman_catalog_response = requests.post(asset_service_url, headers=headers, json=rman_catalog_create_request, verify=False)
+
+    if oracle_create_rman_catalog_response.status_code != 201:
+        print("\nAPI returned status code {}. Response: {}".format(oracle_create_rman_catalog_response.status_code,
+                                                                 oracle_create_rman_catalog_response.json()))
+        raise SystemExit("\nScript ended.\n")
+
+    print ("\nRequest to create RMAN catalog has been posted.")
+
+    oracle_rman_catalog_uri = verify_asset_creation(oracle_create_rman_catalog_response.json()['data']['id'])
+    if oracle_rman_catalog_uri != '':
+        print("\nOracle RMAN Catalog created.")
+        return oracle_rman_catalog_uri.split('/')[-1]
+    else:
+        print("RMAN catalog creation did not respond in time, continuing without mapping the databse to the catalog")
+        return ''
+
+def add_credentials(credentialOption, credentialName):
+    print("Adding Credentials")
+    creds_create_request = open_sample_payload("post_oracle_add_creds.json")
+    creds_create_request['data']['attributes']['name']= credentialName
+    if credentialOption == "wallet":
+        creds_create_request['data']['attributes']['contents'] = {"useWallet": True}
+    elif credentialOption == "oracle" or credentialOption ==  "osAndOracle":
+        print("Enter Oracle Username")
+        oraUser = input()
+        print("Enter Oracle Password")
+        oraPass = input()
+        creds_create_request['data']['attributes']['contents'] = {"oracleUsername": oraUser, "oraclePassword": oraPass}
+    if credentialOption == "os" or credentialOption == "osAndOracle":
+        print("Enter OS Username")
+        creds_create_request['data']['attributes']['contents']['osUser'] = input()
+        print("Enter OS Password")
+        creds_create_request['data']['attributes']['contents']['osPassword'] = input()
+        print("Enter OS Domain")
+        creds_create_request['data']['attributes']['contents']['osDomain'] = input()
+
+    oracle_create_creds_response = requests.post(credentials_url, headers=headers, json=creds_create_request, verify=False)
+
+    if oracle_create_creds_response.status_code != 201:
+        print("\nAPI returned status code {}. Response: {}".format(oracle_create_creds_response.status_code,
+                                                                 oracle_create_creds_response.json()))
+        raise SystemExit("\nScript ended.\n")
+
+    print ("\nCreated creds for Oracle")
+
+def open_sample_payload(filename):
+    cur_dir = os.path.dirname(os.path.abspath(__file__))
+    cur_dir = cur_dir + os.sep + "sample-payloads" + os.sep
+    file_name = os.path.join(cur_dir, filename)

```

Comment:
I think it's peculiar to concatenate with `os.sep` on one line and then use `os.path.join` on the next.

```suggestion
    file_name = os.path.join(cur_dir, 'sample-payloads', filename)
```

With that change, we also don't falsely claim that the sample-payloads directory is the "current" directory.

Reply:
Yeah it was copy paste from another area tbh. I've changed it to the suggestion